webapp/views.py

```diff
@@ -393,6 +394,44 @@ def get_notice(notice_id, **kwargs):
             404,
         )
 
+    releases = {
+        release.codename: release.version
+        for release in db_session.query(Release).all()
+    }
+
+    package_descriptions = {}
+    package_versions = {}
+    release_packages = SortedDict()
+
+    if notice.release_packages:
+        for codename, pkgs in notice.release_packages.items():
+            release_version = releases[codename]
+            release_packages[release_version] = {}
+            for package in pkgs:
+                if not package.get("is_visible", True):
+                    continue
+
+                name = package["name"]
+                if not package["is_source"]:
+                    release_packages[release_version][name] = package
+                    continue
```

Comment:
Could do with a comment to explain what a source package is and why we exclude (?) them. Unless you don't know :D but I guess if you copied the logic from somewhere else you could even just mention that ("I don't know why we exclude source packages, but I followed the logic from ....")